lib/rucio/core/did.py

```diff
@@ -1296,7 +1294,7 @@ def get_did_meta(scope, name, session=None):
     try:
         row = session.query(models.DidMeta).filter_by(scope=scope, name=name).one()
         meta = getattr(row, 'meta')
-        return json.loads(meta) if session.bind.dialect.name == 'oracle' else meta
+        return json.loads(meta) if session.bind.dialect.name == 'oracle' or session.bind.dialect.name == 'sqlite' else meta
```

Comment:
can you try if it works with a statement like:
`.... if session.bind.dialect.name in ['oracle', 'sqlite'] ....`